### PR TITLE
Feature/styling

### PR DESCRIPTION
**Pull request by matovu-farid**
In this branch, the following changes were added:
- The navbar was styled
- The book-list was styled
- The buttons were styled
- A progress bar was added
- The form was styled
- The README.md was updated

### CODE REVIEW

File: src/components/BookItem/ItemStart/ItemStart.css [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+}
+
+.item-start {
+  display: flex;
+  flex-direction: column;
+  align-self: stretch;
+}
+
+.item-main {
+  /* margin-bottom: 20px; */
```
Comment (moise10r) [inline src/components/BookItem/ItemStart/ItemStart.css]: - [ ] I think you have forgotten to remove this commented line, Kindly get rid of it, It is always a good practice to keep your code clean from such comments.

**Review by moise10r — CHANGES_REQUESTED** [review]
### Hi **@matovu-farid,**

You did a great job so far 👏🏼 👏🏼
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

### To highlight 💪🏻

- No linter errors. ✔️ 
- Gitflow was correctly used.  ✔️   
- Good commit messages ✔️ 
 
 👍  Good job!


## STATUS: CHANGES REQUESTED  ♻️ 
_Please Check the comments under the review and make all required changes_

- [ ]  Your README looks great 👍🏻  but I think you can improve it to make it more professional by adding a `project description` to it. please refer to this [template](https://github.com/microverseinc/readme-template).

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏 💻  
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me @moise10r when you comment so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/BookItem/BookItem.css [hunk 2]
```diff
@@ -0,0 +1,10 @@
+
+.item-card {
+  display: flex;
+  justify-content: space-between;
```
Comment (moise10r) [inline src/components/BookItem/BookItem.css:4]: - [ ] [OPTIONAL] This is great using flexbox to layout your content, congrats 👏 , But we can still improve this code by using DRY (don't repeat yourself) but why? well, I realized that you have used the property `display: flex` at multiple places, please let's keep the code clean by extracting the property into a separate class that can be reused at multiple places, see the example below: 

- CSS file
```
.d-flex {
  display: flex;
  justify-content: center;
}
```
- HTML file
```
<div class="d-flex">{your content}</div>
```

And remember you can do this for all the other properties that seem to be repeated multiple times, With this, I promise you'll be coding like a pro 😉

**Review by elyor-doniyorov — APPROVED** [review]
Hi @matovu-farid, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**